controller/linodevpc_controller_helpers.go

```diff
@@ -41,7 +41,7 @@ func (r *LinodeVPCReconciler) reconcileVPC(ctx context.Context, vpcScope *scope.
 	}
 
 	if createConfig.Label == "" {
-		createConfig.Label = util.RenderObjectLabel(vpcScope.LinodeVPC.UID)
```

Comment:
are there other usages of renderObjectLabel? or can this function be deleted?

Reply:
Yeah, my IDE shows no other usages so we should be safe to remove this

Reply:
Deleted the function. It doesn't affect anything else